tensorboard/components/vz_line_chart2/vz-line-chart2.html

```diff
@@ -124,14 +124,16 @@
         cursor: crosshair;
       }
 
-      :host.altdown #chartdiv .main {
-        cursor: default;
+      :host.altdown #chartdiv :not(.drag-zooming) .main,
+      :host #chartdiv .panning .main {
```

Comment:
Strictly speaking this particular case never happens, right?  Once .panning is added, that means you've actually started the pan and .mousedown also applies so the rule below takes precedence.  And when .mousedown ends, panning also ends, so this rule won't take effect then either.

I think this rule only needs to cover the case when alt is pressed and there's no drag-zooming happening.

At this point, it seems perhaps easier to to move the key/mouse event handling logic entirely into the PanZoomDragLayer (it's already partially in there due to the mouse filters).  Then it would be easier to express these rules very directly as just:

```
if panning:
  cursor = grabbing
else if drag-zooming:
  cursor = crosshair
else if altdown:
  cursor = grab
else:
  cursor = crosshair
```